### PR TITLE
Monitor endpoints

### PR DESCRIPTION
**Pull request by frikkf**

### CODE REVIEW

File: src/config/config.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const {
   LIGHTSTEP_COLLECTOR_PORT = '80',
   LIGHTSTEP_COLLECTOR_HOST,
   LIGHTSTEP_ACCESS_TOKEN,
-  COGNITE_API_BASE_URL = 'https://greenfield.cognitedata.com',
+  COGNITE_API_BASE_URL = 'https://api.cognitedata.com',
```
Comment (issaafalkattan) [inline src/config/config.ts]: arent we keeping this greenfield?

File: src/db/database.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ export class PostgresDatasource {
       'db.type': 'pg',
       'db.statement': config.text,
     });
-
+    console.log(config.text);
```
Comment (issaafalkattan) [inline src/db/database.ts]: probably should be removed

File: src/config/config.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ const {
   LIGHTSTEP_COLLECTOR_PORT = '80',
   LIGHTSTEP_COLLECTOR_HOST,
   LIGHTSTEP_ACCESS_TOKEN,
-  COGNITE_API_BASE_URL = 'https://api.cognitedata.com',
+  COGNITE_API_BASE_URL = 'https://greenfield.cognitedata.com',
```
Comment (issaafalkattan) [inline src/config/config.ts]: ah nevermind previous comment im going by commits

File: src/db/database.ts [hunk 4]
```diff
@@ -44,7 +44,6 @@ export class PostgresDatasource {
       'db.type': 'pg',
       'db.statement': config.text,
     });
-    console.log(config.text);
```
Comment (issaafalkattan) [inline src/db/database.ts]: also nevermind old comment

File: src/config/config.ts (hunk 1 above)
Comment (frikkf) [reply src/config/config.ts]: yes thanks ;)

**Pull request merged**